flask_rebar/swagger_generation/swagger_generator_v3.py

```diff
@@ -139,6 +142,11 @@ def _get_paths(self, paths, default_headers_schema, default_security=None):
         for path, methods in paths.items():
             swagger_path, path_args = format_path_for_swagger(path)
 
+            if all(
+                d.hidden and not self.include_hidden for method, d in methods.items()
+            ):
```

Comment:
```suggestion
            if not self.include_hidden and all(
                d.hidden for method, d in methods.items()
            ):
```
Minor streamlining opportunity here since we only need to check `include_hidden` once